cs/amo_policies.md

```diff
@@ -5,7 +5,7 @@ Poslední aktualizace 18. března 2024
 
 [addons.mozilla.org (AMO)](https://addons.mozilla.org/) je online komunita, která hostuje obsah vytvořený uživateli a vývojáři Firefox. Účastí v této komunitě souhlasíte s [Podmínkami používání webových stránek a komunikačních služeb](https://www.mozilla.org/about/legal/terms/mozilla/) společnosti Mozilla a s tím, že se na vás vztahuje [Oznámení o ochraně osobních údajů pro webové stránky, komunikaci a soubory cookie](https://www.mozilla.org/privacy/websites/) společnosti Mozilla.
 
-Můžete se rozhodnout přidat rozšíření, hodnocené, motivy, kolekce nebo jiný obsah od ostatních členů komunity k použití. Současně můžete nahrávat nebo jinak přidávat obsah pro vytváření uživatelského profilu na AMO. [Zásady přijatelného používání](https://www.mozilla.org/about/legal/acceptable-use/) společnosti Mozilla stanoví zásady chování uživatelů ve všech produktech a na všech webových stránkách společnosti Mozilla, a to včetně AMO. Navíc musí být všechna tato rozšíření, motivy a kolekce sdílené na AMO v souladu se [Zásadami pro doplňky](https://extensionworkshop.com/documentation/publish/add-on-policies/) společnosti Mozilla, které zahrnují standardy pro technickou stránku, ochranu soukromí, transparentnost a související standardy pro doplňky.
+Můžete se rozhodnout přidat rozšíření, hodnocení, motivy, kolekce nebo jiný obsah od ostatních členů komunity k použití. Současně můžete nahrávat nebo jinak přidávat obsah pro vytváření uživatelského profilu na AMO.  [Zásady přijatelného používání](https://www.mozilla.org/about/legal/acceptable-use/) společnosti Mozilla stanoví zásady chování uživatelů ve všech produktech a na všech webových stránkách společnosti Mozilla, a to včetně AMO. Navíc musí být všechna tato rozšíření, motivy a kolekce sdílené na AMO v souladu se [Smlouvou o distribuci doplňků Firefox](https://extensionworkshop.com/documentation/publish/firefox-add-on-distribution-agreement/), včetně [Zásad pro doplňky](https://extensionworkshop.com/documentation/publish/add-on-policies/), které upravují standardy pro technickou stránku, ochranu soukromí, transparentnost a související standardy pro doplňky.
```

Comment:
```suggestion
Můžete se rozhodnout přidat rozšíření, hodnocení, motivy, kolekce nebo jiný obsah od ostatních členů komunity k použití. Současně můžete nahrávat nebo jinak přidávat obsah pro vytváření uživatelského profilu na AMO. [Zásady přijatelného používání](https://www.mozilla.org/about/legal/acceptable-use/) společnosti Mozilla stanoví zásady chování uživatelů ve všech produktech a na všech webových stránkách společnosti Mozilla, a to včetně AMO. Navíc musí být všechna tato rozšíření, motivy a kolekce sdílené na AMO v souladu se [Smlouvou o distribuci doplňků Firefox](https://extensionworkshop.com/documentation/publish/firefox-add-on-distribution-agreement/), včetně [Zásad pro doplňky](https://extensionworkshop.com/documentation/publish/add-on-policies/), které upravují standardy pro technickou stránku, ochranu soukromí, transparentnost a související standardy pro doplňky.
```

Reply:
I'm going to fix the translations here. Double spacing after a period is not a style that we follow in American English, but it's definitely not something that should be carried over to localized versions.